Source/BugsnagKSCrashSysInfoParser.m

```diff
@@ -16,9 +16,10 @@
 #define PLATFORM_WORD_SIZE sizeof(void*)*8
 
 NSDictionary *BSGParseDevice(NSDictionary *report) {
-    NSMutableDictionary *device =
-    [[report valueForKeyPath:@"user.state.deviceState"] mutableCopy];
-    
+    NSMutableDictionary *device = [NSMutableDictionary new];
```

Comment:
Would it be better to do something like this? I'm not sure how `initWithDictionary` handles nil

```
NSDictionary *state = [report valueForKeyPath:@"user.state.deviceState"];
NSMutableDictionary* device = [[NSMutableDictionary alloc] initWithDictionary: state];
```

Reply:
`initWithDictionary:` requires a Nonnull parameter, whereas `addEntriesFromDictionary:` handles a nil param.